popde/density_estimate.py

```diff
@@ -1,3 +1,128 @@
+class SimpleGaussianKernelDensityEstimation:
+    """
+    Kernel Density Estimation (KDE) using Gaussian Kernels
+
+    Parameters:
+    -----------
+    data : array-like
+        The input data for which KDE will be estimated.
+
+    bandwidth : float, optional (default=1.0)
+        Bandwidth parameter controlling the width of the kernels.
+
+    kernel : str, optional (default='gaussian')
+        The kernel function used for smoothing. Supported kernels: 'gaussian',
+
+    Attributes:
```

Comment:
I don't know if it is worth writing all of the attributes in a class docstring. Is this recommended anywhere?

Reply:
removed